setuptools/command/bdist_wheel.py

```diff
@@ -595,3 +593,16 @@ def adios(p: str) -> None:
             shutil.copy(license_path, os.path.join(distinfo_path, filename))
 
         adios(egginfo_path)
+
+    def _zip_compression(self) -> int:
+        if (
+            isinstance(self.compression, int)
+            and self.compression in self.supported_compressions.values()
+        ):
+            return self.compression
+
+        compression = self.supported_compressions.get(self.compression)
+        if compression is not None:
+            return compression
```

Comment:
Ok, so let's walrus our way out of it (I can see that the walrus is used in `distutils` and other code bases maintained by Jason).

```suggestion
        if isinstance(self.compression, str) and (
            compression := self.supported_compressions.get(self.compression)
        ):
            return compression
```